packages/blockchain-common/test/registries/FabricWalletRegistry.test.ts

```diff
@@ -13,21 +13,22 @@
 */
 
 import * as chai from 'chai';
-import { TestUtil } from '../TestUtil';
-import { FabricWalletRegistryEntry } from '../../extension/registries/FabricWalletRegistryEntry';
-import { FabricWalletRegistry } from '../../extension/registries/FabricWalletRegistry';
-import { FabricRuntimeManager } from '../../extension/fabric/FabricRuntimeManager';
-import { FabricWalletUtil } from '../../extension/fabric/FabricWalletUtil';
+import * as path from 'path';
+import * as chaiAsPromised from 'chai-as-promised';
+import { FabricWalletRegistryEntry } from '../../src/registries/FabricWalletRegistryEntry';
+import { FabricWalletRegistry } from '../../src/registries/FabricWalletRegistry';
+import { FabricWalletUtil } from '../../src/util/FabricWalletUtil';
 
+chai.use(chaiAsPromised);
 chai.should();
 
 describe('FabricWalletRegistry', () => {
 
     const registry: FabricWalletRegistry = FabricWalletRegistry.instance();
 
     before(async () => {
-        await TestUtil.setupTests();
-        await FabricRuntimeManager.instance().getRuntime().create();
+        const registryPath: string = path.join(__dirname, 'tmp/registries');
```

Comment:
`path.join(__dirname, './tmp/registries')` should be `path.join(__dirname, 'tmp', 'registries')`